scripts/install/openvstorage-webapps.postinst.py

```diff
@@ -77,9 +77,9 @@
 
 # Setup nginx site
 if not os.path.exists("/etc/nginx/sites-enabled/openvstorage.conf"):
-    check_output("ln -s /etc/nginx/sites-available/openvstorage.conf /etc/nginx/sites-enabled/openvstorage.conf", shell=True)
+    check_output("ln -sf /etc/nginx/sites-available/openvstorage.conf /etc/nginx/sites-enabled/openvstorage.conf", shell=True)
```

Comment:
Why to force override of existing files if this code is in `not os.path.exists("/etc/nginx/sites-enabled/openvstorage.conf")` condition?